test/onnx/test_fx_to_onnx.py

```diff
@@ -434,25 +434,6 @@ def test_fake_tensor_mode_huggingface_gpt2(self):
             onnx.checker.check_model(export_output.model_proto)
             onnx.shape_inference.infer_shapes(export_output.model_proto)
 
-    def test_fake_tensor_mode_huggingface_bigscience_bloom(self):
```

Comment:
it seems this one was deleted instead of moved?

Reply:
I already have bloom in runtime.